app/controllers/api/v3/declarations_controller.rb

```diff
@@ -1,10 +1,35 @@
 module API
   module V3
     class DeclarationsController < BaseController
+      include Pagination
+      include ::API::Concerns::FilterByUpdatedSince
+
+      def index
+        render json: to_json(paginate(declarations_query.declarations))
+      end
+
       def create = head(:method_not_allowed)
       def show = head(:method_not_allowed)
-      def index = head(:method_not_allowed)
       def void = head(:method_not_allowed)
+
+    private
+
+      def declarations_query
+        conditions = { lead_provider: current_lead_provider, updated_since:, participant_ids:, cohort_start_years: }
```

Comment:
I think we also need `delivery_partner_id` on v3

Reply:
is delivery partner not just ECF though?

Reply:
🤦 yep!